.github/workflows/changed_file_trigger_examples_check.yml

```diff
@@ -0,0 +1,109 @@
+name: Test Example
+on:
+  pull_request:
+    paths:
+      - 'examples/**'
+    # So only the changes in examples folder will trigger jobs below.
+  schedule:
+    # run at 00:00 of every Sunday(singapore time) so here is UTC time Saturday 16:00
+    - cron:  '0 16 * * 6'
+
+jobs:
+# This is for changed files detect and check.
+  detect-changed-files:
+    if: ${{ github.event_name == 'pull_request' }}
```

Comment:
Add some more conditions to ensure this workflow is only executed in the organization's repo instead of any forked repo, refer to this [workflow](https://github.com/hpcaitech/ColossalAI/blob/main/.github/workflows/build.yml) for details.

`github.base_ref == 'main' && github.event.pull_request.base.repo.full_name == 'hpcaitech/ColossalAI'`

Reply:
Done!